### PR TITLE
chore: bump code-block, resolve related TODO

### PR DESCRIPTION
**Pull request by zchsh**
## "Ready for Review" checklist

- [x] The Vercel preview link has been added to this PR's description
- [x] The Asana task has been added to this PR's description
- [x] This PR's link has been added to the "📐 Pull Request" field on the Asana task
- [x] This Vercel preview link has been added to the "🔍 Preview" field on the Asana task
- [x] The description template has been filled in below

---

## Relevant links

- [Preview link][waypoint/downloads] 🔎
- [Asana task](https://app.asana.com/0/1100423001970639/1201881376116200/f) 🎟️

## What

This PR bumps `@hashicorp/react-code-block` to pull in a bug fix, and a tabs alignment change, as implemented in https://github.com/hashicorp/react-components/pull/532.

## Why

- Fixes a bug that was preventing use of `CodeTabs` on "Install" views
- Updates tabs to be left-aligned when no `heading` is present, to match a design ask

## How

Bumps a dependency, work was mainly done in the the `code-block` package in https://github.com/hashicorp/react-components/pull/532.

## Testing

- [ ] Visit the install views, ensure `code-tabs` are rendering as expected
    - [ ] [`/waypoint/downloads`][waypoint/downloads]

Compare to the same page prior to bump the dependency:

- [ ] Visit <https://dev-portal-6dwd3gxpl-hashicorp.vercel.app/waypoint/downloads>
    - The page built successfully, and works fine for static builds...
    - But it fails with ISR, as it fails to re-hydrate on the client due to the (inaccurate) `CodeTabs` error, due to now-minified function names.

## Anything else?

Nope!

[waypoint/downloads]: https://dev-portal-git-zsbump-code-block-hashicorp.vercel.app/waypoint/downloads

### CODE REVIEW

**Comment by zchsh** [comment]
@ashleemboyer This rolls out the work you recently reviewed in https://github.com/hashicorp/react-components/pull/532. Let me know if this looks alright! (Definitely still lots of work to do in updating the look and feel of `@hashicorp/react-code-block`... but that' a whole separate thing 😅 )

File: src/views/product-downloads-view/components/downloads-section/index.tsx [hunk 1]
```diff
@@ -57,20 +58,19 @@ const PackageManagerSection = ({ packageManagers, prettyOSName }) => {
          *
          * ref: https://app.asana.com/0/1201010428539925/1201881376116200/f
          */
-        <Tabs showAnchorLine={false}>
+        <CodeTabs tabs={packageManagers.map(({ label }) => label)}>
```
Comment (ashleemboyer) [inline src/views/product-downloads-view/components/downloads-section/index.tsx:61]: GitHub won't let me make a comment on the `TODO`, but that comment can be removed now! 🎉

**Review by ashleemboyer — CHANGES_REQUESTED** [review]

File: src/views/product-downloads-view/components/downloads-section/index.tsx (hunk 1 above)
Comment (zchsh) [reply src/views/product-downloads-view/components/downloads-section/index.tsx:55]: dang it ahah i thought i did this - thanks for catching it - removed! ✅

**Review by ashleemboyer — APPROVED** [review]
🌟

**Pull request merged**